<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>516</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LYLzip</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="table_file_info">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>258</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>文件信息</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </row>
      <column>
       <property name="text">
        <string>文件名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件大小</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件路径</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_file_comment">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>压缩文件注释</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="text_file_comment">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>258</width>
        <height>10</height>
       </size>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_compress">
        <property name="text">
         <string>压缩</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_decompress">
        <property name="text">
         <string>解压</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_recent_files">
     <property name="title">
      <string>最近文件</string>
     </property>
    </widget>
    <addaction name="menu_recent_files"/>
    <addaction name="separator"/>
    <addaction name="act_open_file"/>
    <addaction name="act_close_file"/>
    <addaction name="separator"/>
    <addaction name="act_save_file_as"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menu_eidt">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="act_add_cmpfile_comment"/>
    <addaction name="act_delete_cmpfile_comment"/>
   </widget>
   <widget class="QMenu" name="menu_code">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="act_encode"/>
    <addaction name="act_decode"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_eidt"/>
   <addaction name="menu_code"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_open_file">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="act_close_file">
   <property name="text">
    <string>关闭文件</string>
   </property>
   <property name="toolTip">
    <string>关闭文件</string>
   </property>
  </action>
  <action name="act_save_file_as">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>将当前文件另存为其他目录</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本程序</string>
   </property>
  </action>
  <action name="act_add_cmpfile_comment">
   <property name="text">
    <string>添加注释</string>
   </property>
   <property name="toolTip">
    <string>为压缩文件添加注释</string>
   </property>
  </action>
  <action name="act_delete_cmpfile_comment">
   <property name="text">
    <string>删除注释</string>
   </property>
   <property name="toolTip">
    <string>将压缩文件中的注释删除</string>
   </property>
  </action>
  <action name="act_encode">
   <property name="text">
    <string>加密文件</string>
   </property>
   <property name="toolTip">
    <string>对任意格式的文件进行加密</string>
   </property>
  </action>
  <action name="act_decode">
   <property name="text">
    <string>解密文件</string>
   </property>
   <property name="toolTip">
    <string>将加密后的文件解密</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
